app/views/membership.html

```diff
@@ -0,0 +1,292 @@
+                    class="meta">
+                    Learn more <i class="fa fa-external-link"></i>
+                  </a>
+                </p>
+              </div>
+              <div column class="content-pane content-{{subjectKind.name.toLowerCase()}}">
+
+                <div
+                  class="col-heading item-row"
+                  row mobile="column" tablet="column">
+                  <div row flex>
+                    <div class="col-name">
+                      <h3>Name</h3>
+                    </div>
+                    <div class="col-roles" flex>
+                      <h3>Roles</h3>
+                    </div>
+                  </div>
+                  <div class="col-add-role">
+                    <h3
+                      ng-show="mode.edit"
+                      class="hidden-xs">
+                      Add another role
+                    </h3>
+                    <a
+                      href=""
+                      ng-if="'rolebindings' | canI : 'update'"
+                      ng-show="!(mode.edit)"
+                      ng-click="mode.edit = !mode.edit">
+                      Edit {{ (subjectKind.name | startCase).toLowerCase() }} membership
+                    </a>
+                  </div>
+                </div>
+
+                <div ng-if="(subjectKind.subjects | hashSize) === 0">
+                  <p>
+                    <em>There are no {{ (subjectKind.name | startCase).toLowerCase() }}s with access to this project.</em>
+                  </p>
+                </div>
+                <div
+                  ng-repeat="(subjectName, subject) in subjectKind.subjects"
+                  class="item-row"
+                  row>
+                  <div class="col-name">
+                    <a
+                      ng-show="subject.namespace"
+                      target="_blank"
+                      ng-href="project/{{project.metadata.name}}/browse/other?kind=ServiceAccount">
+                      <span>
+                        {{subject.namespace}}:
+                      </span>
+                      <strong>
+                        {{subject.name}}
+                      </strong>
+                    </a>
+                    <span ng-show="!subject.namespace">
+                      <strong>
+                        {{subject.name}}
+                      </strong>
+                      <span ng-if="subject.name === user.metadata.name">
+                        (you)
+                      </span>
+                    </span>
+                  </div>
+                  <div
+                    class="action-set"
+                    flex row tablet="column" mobile="column">
+                    <div
+                      class="col-roles"
+                      row tablet="column" flex wrap axis="start">
+                      <action-chip
+                        ng-repeat="role in subject.roles track by $index"
```

Comment:
why are you tracking by $index, that means that the role in position 1 in the array should always be considered the same object from a data model perspective, I don't think that is accurate here.  Seeing this several places in this file.